src/Test.UnitTests.Sarif/Core/SarifLogTests.cs

```diff
@@ -302,62 +302,69 @@ private static SarifLog GetSarifLogWithMinimalUniqueData()
         }
 
         [Fact]
-        public async Task SarifLog_Post_WithValidParameters_ShouldNotThrownAnExceptionWhenRequestIsValid()
```

Comment:
>SarifLog_Post_WithValidParameters_ShouldNotThrownAnExceptionWhenRequestIsValid [](http://example.com/codeflow?start=26&length=78)

These tests were poorly factored and as a result made it difficult to locate an underlying test helper bug. It's a good object lesson for why we factor tests the way that we do, it's to lower costs in resolving regressions (in this case introduced by an update to net6.0 from netcoreapp3.1).